client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -135,6 +135,69 @@ function mediaButton( editor ) {
 		return { isLoaded, onLoad };
 	} )();
 
+	function parseShortcode( content ) {
```

Comment:
I'd be wary of copying this over. Is it possible to trigger a resize with an increment of 0? https://github.com/Automattic/wp-calypso/blob/master/client/components/tinymce/plugins/media/plugin.jsx#L387